tests/iteminfo_test.cpp

```diff
@@ -1285,7 +1285,7 @@ TEST_CASE( "armor protection", "[iteminfo][armor][protection]" )
         item super_tanktop( "test_complex_tanktop" );
         REQUIRE( super_tanktop.get_covered_body_parts().any() );
         // these values are averaged values but test that assumed armor portion is working at all
-        expected_armor_values( super_tanktop, 14, 18, 14.4f, 10.66667f );
+        expected_armor_values( super_tanktop, 11, 18, 14.4f, 10.66667f );
```

Comment:
```suggestion
        expected_armor_values( super_tanktop, 14, 14, 11.2f, 8.66667f );
```

Also (I can't suggest these changes because they're out of range), some of the values on the following lines need to be adjusted:
```diff
diff --git a/tests/iteminfo_test.cpp b/tests/iteminfo_test.cpp
index 686c797af3..d8e3483cbb 100644
--- a/tests/iteminfo_test.cpp
+++ b/tests/iteminfo_test.cpp
@@ -1295,8 +1295,8 @@ TEST_CASE( "armor protection", "[iteminfo][armor][protection]" )
                "  Default:  <color_c_yellow>100</color>\n"
                "<color_c_white>Torso Protection</color>: <color_c_red>4%</color>, <color_c_yellow>Median</color>, <color_c_green>4%</color>\n"
                "  Bash:  <color_c_red>1.00</color>, <color_c_yellow>11.00</color>, <color_c_green>21.00</color>\n"
-               "  Cut:  <color_c_red>1.00</color>, <color_c_yellow>11.00</color>, <color_c_green>27.00</color>\n"
-               "  Ballistic:  <color_c_red>1.00</color>, <color_c_yellow>7.00</color>, <color_c_green>16.00</color>\n" );
+               "  Cut:  <color_c_red>1.00</color>, <color_c_yellow>11.00</color>, <color_c_green>21.00</color>\n"
+               "  Ballistic:  <color_c_red>1.00</color>, <color_c_yellow>7.00</color>, <color_c_green>13.00</color>\n" );
     }
```

Basically:
- `27.00` -> `21.00`
- `16.00` -> `13.00`